packages/website/src/components/ShowcaseLink.astro

```diff
@@ -0,0 +1,21 @@
+---
+import { ArrowRightIcon } from '../icons/heroicons';
+const { app, repo, name } = Astro.props;
+let showcase;
+
+if (app !== undefined) {
```

Comment:
I think we're meant to show both the repo link and the live app link for these (mentioned in my overall comment).

Reply:
I had asked for the showcase information and was given the GitHub showcase repo, I just realized they actually have live apps for those.  🤦‍♂️ I will get that added tomorrow.